jupyter_console/ptshell.py

```diff
@@ -780,11 +834,29 @@ def handle_iopub(self, msg_id=''):
                         if 'text/plain' in data:
                             print(data['text/plain'])
 
+                # If execute input: print it
                 elif msg_type == 'execute_input':
                     content = sub_msg['content']
+
+                    # New line
+                    sys.stdout.write('\n')
+                    sys.stdout.flush()
+
+                    # With `[remote]` prefix
                     if not self.from_here(sub_msg):
-                        sys.stdout.write(self.other_output_prefix)
-                    sys.stdout.write('In [{}]: '.format(content['execution_count']))
+                        print_formatted_text(FormattedText([
+                            ('#ffff00', self.other_output_prefix)]), end='')
```

Comment:
We should look into providing full syntax highlighting for code send over from other clients.

Reply:
I lazily waited for some hint of future integration before doing so. a5a291d

By default:

![jupyter_async_color_default](https://user-images.githubusercontent.com/6123962/73138471-9e3bea00-4041-11ea-8ca2-1e915cc5dd32.png)

With the most terrible color imaginable (configuration after):

![jupyter_async_color](https://user-images.githubusercontent.com/6123962/73138476-ab58d900-4041-11ea-91ef-ffd1fc2efb13.png)



my `~/.jupyter/jupyter_console_config.py` (for testing this commit):
```python
from pygments.token import Token

# See other output
c = get_config()
c.ZMQTerminalInteractiveShell.include_other_output = True

c.ZMQTerminalInteractiveShell.highlighting_style_overrides = {
    Token.Prompt: '#2aa198',  # cyan
    Token.PromptNum: '#268bd2 bold',  #blue
    Token.OutPrompt: '#b58900',  # yellow
    Token.RemotePrompt: '#d33682',  # magenta
}
```